lib/logstash/filters/urldecode.rb

```diff
@@ -14,7 +14,10 @@ class LogStash::Filters::Urldecode < LogStash::Filters::Base
   # Urldecode all fields
   config :all_fields, :validate => :boolean, :default => false
 
-  # Thel character encoding used in this filter. Examples include `UTF-8`
+  # Urldecode all fields starting with prefix
+  config :prefix, :validate => :string
```

Comment:
I see a potential confusion with this two config options, while I like the idea of having this kind if filter, not sure what is the best option to configure it. I see different options.

* We change the ```all_fields```  to be ```string``` so it can include a pattern, but then not sure how to properly express the all fields selection, might be with ```*```. This would be a breaking change.
* We keep it like is now, but we should add probably a check to see when boths options are setup, or add a warning to make the user aware of setting up ```all_fields``` and ```prefix```. This would be probably the best and easy path for now.

what do you think?